apps/router-e2e/__e2e__/server/app/methods+api.ts

```diff
@@ -0,0 +1,44 @@
+import { ExpoResponse } from 'expo-router/server';
+
+/** @type {import('expo-router/server').RequestHandler} */
+export function GET() {
```

Comment:
Maybe also documenting/testing this one could help users with TS.

```suggestion
import { ExpoResponse, type RequestHandler } from 'expo-router/server';

export const GET: RequestHandler = () => {
```

Reply:
This should be covered in the `babel-preset-expo` tests.